### PR TITLE
write schema for pet types table

### PR DESCRIPTION
**Pull request by kducasse**
Just testing to see how pull requests work.

### CODE REVIEW

**Comment by jsim0809** [comment]
Looks good!

File: db/schema.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE pet_types (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR NOT NULL,
+  description TEXT
```
Comment (jsim0809) [inline db/schema.sql:5]: What does TEXT do?

**Comment by MtnGreen** [comment]
Yay - it's working!

File: db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE pet_types (
+  id SERIAL PRIMARY KEY,
```
Comment (MtnGreen) [inline db/schema.sql:2]: Hey - nice SERIAL PRIMARY KEY!

File: db/schema.sql (hunk 1 above)
Comment (kducasse) [reply db/schema.sql:5]: TEXT is like VARCHAR except it doesn't have a character limit.

File: db/schema.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE pet_types (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR NOT NULL,
```
Comment (jsim0809) [inline db/schema.sql]:
```suggestion
  type VARCHAR(50) NOT NULL,
  
```

**Review by jsim0809 — CHANGES_REQUESTED** [review]
Suggesting this change to save space.

**Review by jsim0809 — APPROVED** [review]

**Review by DanAlberts — APPROVED** [review]

**Pull request merged**